### DISCUSSION TITLE
10.9.1 Release

### DISCUSSION THREAD

**Discussion by yhaskell** [cssinjs/jss#1615] [Q&A] [answered]
Hello there!

As for now, 10.9.1 was in the alpha state for 22 days without changes.

I would like to know, what is the release mechanism?
For example, is there some rule, such as "release will happen 30 days after alpha was available"?

Or, is there anything I, or other people, can contribute, to facilitate the release of the final version?

Thanks in advance!

**Comment by kof** [comment, marked as answer]
just waiting random amount of time to see if there are new big bugs, if there are none I can release a stable one any time

**Reply by karlavetyan** [reply]
Hi @kof, maybe any updates?

**Reply by kof** [reply]
we can release it I guess as a non-alpha

**Comment by kof** [comment]
10.9.1 released
